team_project/src/webapp/offerForm.css

```diff
@@ -0,0 +1,35 @@
+.navbar{
```

Comment:
Minor suggestion: since this css is used by both requestForm and offerForm, maybe just call it form.css?